.gitlab-ci.yml

```diff
@@ -140,12 +141,12 @@ build-rust-doc-release:            &build
     expire_in:                     7 days
     paths:
     - ./crate-docs
-  <<:                              *build-only
+  # <<:                              *build-only
```

Comment:
if there is no only statement the doc build will run every time. If that is desired why not have it consistent with *test-linux-stable*?

Reply:
it was just for the test run, won't be in the final commit.